sdk/core/Azure.Core/api/Azure.Core.net5.0.cs

```diff
@@ -194,7 +194,9 @@ public partial class RequestContext
         public RequestContext() { }
         public System.Threading.CancellationToken CancellationToken { get { throw null; } set { } }
         public Azure.ErrorOptions ErrorOptions { get { throw null; } set { } }
+        public bool HasCustomClassifier { get { throw null; } }
```

Comment:
The name is also strange to see on classifier. The user will wonder what it means for a classifier to have a classifier.